instrumentation/apolloconfig/apolloconfig-1.0.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apolloconfig/v1_0_0/ApolloConfigInstrumentationModule.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apolloconfig.v1_0_0;
+
+import static java.util.Collections.singletonList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class ApolloConfigInstrumentationModule extends InstrumentationModule {
+  public ApolloConfigInstrumentationModule() {
+    super("apolloconfig", "apolloconfig-1.0");
```

Comment:
I think naming the module `apollo-client` would be better

Reply:
apollo-client reminds people of graphql or autonomous driving. so use their group name apolloconfig

Reply:
We usually use the name of the enhanced entity as the name of the module. If there is ambiguity, we can add the name of the enhanced entity's organization as a prefix